services/merak-compute/activities/vm_info_activity.go

```diff
@@ -49,6 +49,7 @@ func VmInfo(ctx context.Context) (*pb.ReturnComputeMessage, error) {
 			SubnetId:        common.RedisClient.HGet(ctx, vmID, "subnetID").Val(),
 			DefaultGateway:  common.RedisClient.HGet(ctx, vmID, "gw").Val(),
 			Host:            common.RedisClient.HGet(ctx, vmID, "hostname").Val(),
+			RemoteId:        common.RedisClient.HGet(ctx, vmID, "remoteID").Val(),
```

Comment:
@phudtran What is the `remoteID`?

Reply:
Alcor PortID

Reply:
Why not using PortID?

Reply:
Mainly to be more general (In other test targets such as Cilium the term Endpoint is used instead of Port), and to show that this ID belongs to the test target.